client/src/styles/Geocoder.css

```diff
@@ -0,0 +1,79 @@
+.leaflet-left {
+  position: fixed;
+  top: 70%;
+  right: 1%;
+}
+
+.leaflet-right {
+  margin: 1rem 0;
+}
+.leaflet-bar {
+  border-radius: 0.8rem;
+}
+
+.leaflet-control-geocoder-form {
+  display: block;
+  padding: 0.82rem;
+  border: 2px solid var(--secondary-color);
+  padding: 1.2rem 0.5rem 1rem 2rem;
+  border-radius: 0.6rem;
+}
```

Comment:
You can change the code to something like : 
`.leaflet-control-geocoder-form {
  display: block;
  /* padding: 0.82rem; */
  padding: 1rem;
  border: 2px solid var(--secondary-color);
  /* padding: 1.2rem 0.5rem 1rem 2rem; */
  border-radius: 0.6rem;
}`

Reply:
done